src/heads-up/components/app-sidebar.vue

```diff
@@ -95,4 +105,30 @@ export default {
     height: 1em;
     fill: currentColor;
   }
+
+  .sidebar__socials {
```

Comment:
I don't think a `margin-bottom` is necessary. You can also achieve this by setting a higher padding-top value on the `.sidebar__footer`.